apiserver/worker/compiler.py

```diff
@@ -408,7 +408,13 @@ def compile(self, bot_dir, globs, errors, timelimit):
     ),
     Language("Dart", BOT +".dart", "MyBot.dart",
         "dart MyBot.dart",
-        [], [(["*.dart"], ChmodCompiler("Dart"))]),
+        [], [(["*.dart"], ChmodCompiler("Dart"))]
+    ),
+    Language("Elixir", BOT +".ex", "MyBot.ex",
+        "./MyBot",
```

Comment:
This setup requires that the Elixir bot always have a shebang line; is that intentional? Otherwise it might make more sense to have the command executed be `elixir MyBot` or something like that.

Reply:
@lidavidm good question, currently the way the elixir starter kit works is that it uses `escript` to generate an elixir cli interface as its own stand-along executable.  So we can just run it as `./Mybot`as shown in this example [here](https://github.com/HaliteChallenge/Halite-II/blob/master/airesources/Elixir/run_game.sh#L3) but maybe that PR was merged in too hastily.  I'm glad to rework that starter kit to depend on running the bot via the `elixir MyBot.exs` if that's preferable.  Thanks for the feedback and I look forward to your response.

Reply:
Ah, I didn't know that. In that case, this looks good - I'll approve it. I'll try running that bot later and just make sure it's all working. One note though, there is a merge conflict (another package got added in the meantime), if you could address that.

Reply:
So @joelbyler , is the Elixir starter kit able to be zipped up and submitted as-is? There isn't any `MyBot.ex` - or should `elixirbot` be renamed to `MyBot.ex` first?

Reply:
(The `Language` part specifies that for Elixir, we look for a `MyBot.ex` file, run `chmod` on it, then look for a `BOT + '.ex' == 'MyBot.ex'` file to take as the actual bot. It then writes `./MyBot` to a bash script used to start the bot, so right now a few things don't line up.)

Reply:
Anyways, once the bug with `ChmodCompiler` is fixed, it compiles fine if `elixirbot` is renamed `MyBot.ex`.

Reply:
@lidavidm interesting, this has helped me to understand a bit more about how this works now.  I'm going to try and make a new revision to see if I can make this more straight forward.  Thanks a bunch!